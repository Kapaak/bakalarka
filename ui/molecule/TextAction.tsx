import { HorizontalStack, Link, Text } from "..";

interface TextActionProps {
  text: string;
  action: string;
  href: string;
}

export const TextAction = ({ action, href, text }: TextActionProps) => {
  return (
    <HorizontalStack className="gap-2">
      <Text className="font-light" size="small">
        {text}
      </Text>
      <Link href={href}>
        <Text color="secondary" size="small">
          {action}
        </Text>
      </Link>
    </HorizontalStack>
  );
};
