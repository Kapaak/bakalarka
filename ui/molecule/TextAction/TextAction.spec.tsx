import { render } from "@testing-library/react";

import { TextAction } from "./TextAction";

describe("TextAction", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<TextAction text="" action="" href="" />);

    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
