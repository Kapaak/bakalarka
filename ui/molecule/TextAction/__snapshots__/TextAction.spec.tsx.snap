// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextAction should render successfully 1`] = `
<body>
  <div>
    <div
      class="flex justify-normal items-stretch gap-2"
    >
      <p
        class="text-black text-sm font-light"
      />
      <a
        class="relative"
        href=""
      >
        <p
          class="text-main-orange text-sm"
        />
      </a>
    </div>
  </div>
</body>
`;
